rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -441,6 +448,20 @@ protected TxType getTransactionType(BtcTransaction btcTx) {
         return TxType.UNKNOWN;
     }
 
+    private boolean txIsFromOldFederation(BtcTransaction btcTx) {
+        Address oldFederationAddress = Address.fromBase58(bridgeConstants.getBtcParams(), bridgeConstants.getOldFederationAddress());
```

Comment:
Shouldn't this check for an oldFederation in storage and only check the bridge constant if it does not exist?